common/src/main/java/net/sourceforge/swing/ScreenFinder.java

```diff
@@ -34,10 +34,9 @@
 this exception to your version of the library, but you are not
 obligated to do so.  If you do not wish to do so, delete this
 exception statement from your version. */
-package net.sourceforge.jnlp.util;
+package net.sourceforge.swing;
```

Comment:
Maybe we can directly move it in net.adoptopenjdk.... packages?

Reply:
done